BuyPolish/Pola/Utilities/Categories/UIDevice+Additions.swift

```diff
@@ -9,7 +9,9 @@ extension UIDevice {
         var deviceInfo = "\n\n-------App & Device info--------\n"
         deviceInfo.append("- system: \(systemName), \(systemVersion)\n")
         let bundle = Bundle.main
-        deviceInfo.append("- app: \(String(describing: bundle.version)), \(String(describing: bundle.shortVersion))\n")
+        if let version = bundle.version, let shortVersion = bundle.shortVersion {
+            deviceInfo.append("- app: \(String(describing: version)), \(String(describing: shortVersion))\n")
```

Comment:
`version` i `shortVersion` są typu String, nie jest więc już potrzebne owijanie je w `String(describing: XXX)` tak jak było to w przypadku Optionala.